modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheIoManager.java

```diff
@@ -1206,8 +1206,6 @@ public void send(ClusterNode node, GridCacheMessage msg, byte plc) throws Ignite
 
         while (cnt <= retryCnt) {
```

Comment:
Let's replace with `while (true)` and have only 1 exit by the condition in exception handling